src/libraries/System.Configuration.ConfigurationManager/src/Resources/Strings.resx

```diff
@@ -664,4 +717,34 @@
   <data name="DuplicateFileName" xml:space="preserve">
     <value>The file name '{0}' was already in the collection.</value>
   </data>
-</root>
```

Comment:
[nit] I would revert all changes above this line - the addition is redundant and few removed comments look useful.